src/screens/CreatePost/elements/SheetExpiredPost.js

```diff
@@ -4,53 +4,54 @@ import {StyleSheet, Text, View} from 'react-native';
 import RBSheet from 'react-native-raw-bottom-sheet';
 
 import FlatListItem from '../../../components/FlatListItem';
-import {colors} from '../../../utils/colors';
 import {fonts} from '../../../utils/fonts';
+import {COLORS} from '../../../utils/theme';
 
-const SheetExpiredPost = ({refExpired, data, select, onSelect}) => (
-    <RBSheet
-      ref={refExpired}
-      closeOnDragDown={true}
-      closeOnPressMask={true}
-      customStyles={{
-        container: styles.containerSheet,
-        draggableIcon: styles.draggableIcon,
-      }}>
-      <View style={styles.container}>
-        <Text style={styles.header}>Post expiration after</Text>
-        {data.map((value, index) => (
-          <FlatListItem
-            key={index}
-            value={value.label}
-            index={index}
-            select={select}
-            onSelect={onSelect}
-          />
-        ))}
-      </View>
-    </RBSheet>
-  );
+const SheetExpiredPost = ({refExpired, data, select, onSelect, isAnonimity = false}) => (
+  <RBSheet
+    ref={refExpired}
+    closeOnDragDown={true}
+    closeOnPressMask={true}
+    customStyles={{
+      container: styles.containerSheet,
+      draggableIcon: styles.draggableIcon
+    }}>
+    <View style={styles.container}>
+      <Text style={styles.header}>Post expiration after</Text>
+      {data.map((value, index) => (
+        <FlatListItem
+          key={index}
+          value={value.label}
+          index={index}
+          select={select}
+          onSelect={onSelect}
+          isAnonimity={isAnonimity}
+        />
+      ))}
+    </View>
+  </RBSheet>
+);
 
 export default SheetExpiredPost;
 
 const styles = StyleSheet.create({
   container: {
     paddingTop: 20,
-    paddingBottom: 38,
+    paddingBottom: 38
   },
   header: {
-    color: colors.black,
+    color: COLORS.black,
     fontFamily: fonts.inter[600],
     fontSize: 18,
     fontWeight: 'bold',
-    paddingHorizontal: 20,
+    paddingHorizontal: 20
   },
   containerSheet: {
     borderTopRightRadius: 20,
     borderTopLeftRadius: 20,
-    height: 'auto',
+    height: 'auto'
   },
   draggableIcon: {
-    backgroundColor: colors.alto,
-  },
+    backgroundColor: COLORS.alto
+  }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color value for the `draggableIcon` style has been updated to use the `COLORS.alto` constant from the theme file, replacing the previous `colors.alto`. This change improves consistency and maintainability by using centralized color definitions.
```diff
- backgroundColor: colors.alto,
+ backgroundColor: COLORS.alto
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->